src/components/PopupModal/PopupModal.tsx

```diff
@@ -128,7 +128,7 @@ export function PopupModal(props: IPopupModalProps) {
                   overflow-x-hidden overflow-y-auto`}
       {...rest}
     >
-      <div className="relative  max-h-full w-1/2">
+      <div className="relative  max-h-full w-1/2 min-w-[745]">
```

Comment:
Do we really need a min **w**!? 🤔  Weird that it does not adapt to your content...

Have you try to set a **w-...** on the content you are trying to render inside!? 🤔

Reply:
yeah, w-fit and w-full just end up breaking the design on resize bc we didn't make it responsive

Reply:
![Screenshot from 2023-06-06 17-43-40](https://github.com/massalabs/ui-kit/assets/90157528/726dd419-e460-44fa-823d-ba4f257c3fc3)

w-fit minimizes the prob but creates another one